SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Reporting security issues
+
+If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of Grafana Labs's open source and commercial products (including but not limited to Grafana, Grafana Cloud, Grafana Enterprise, and grafana.com). We can accept only vulnerability reports at this address.
```

Comment:
I don't know what was meant by the parenthetical "including but not limited to ... grafana.com" part of this paragraph.  "grafana.com" is not a commercial or open source product, so it does not make sense to me.  Perhaps "grafana.com" should be changed to something like "other products listed at grafana.com".

I'm including this suggestion such that you can see grammar corrections that should be changed.

```suggestion
If you think you have found a security vulnerability, please send a report to [security@grafana.com](mailto:security@grafana.com). This address can be used for all of Grafana Labs' open source and commercial products (including, but not limited to Grafana, Grafana Cloud, Grafana Enterprise, and grafana.com). We accept only vulnerability reports at this address.
```